GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -572,5 +615,77 @@ private void saveAsToolStripMenuItem_Click(object sender, EventArgs e)
                 }
             }
         }
+
+        private string GetSelectedFileInFileTree()
+        {
+            var item = tvGitTree.SelectedNode.Tag as GitItem;
+
+            if (item == null || item.ObjectType == GitObjectType.Tree)
+                return null;
+
+
+            var filename = Path.Combine(Module.WorkingDir, item.FileName);
+            if (!File.Exists(filename))
+            {
+                MessageBox.Show(string.Format("The selected file '{0}' no more exists in the working directory." + Environment.NewLine + "No action could be made on it.", filename), "File not found", MessageBoxButtons.OK, MessageBoxIcon.Warning);
```

Comment:
Few issues:
* this text should be localizable
* the grammar is suffering a little bit, I'd make it shorter to say: _'{0}' no longer exists in the working directory._ (btw have you checked how a message with a very long path (200 symbols+) looks?)

I propose to rephrase the error:

Heading: _Unable to perform requested action_
Message: _'{0}' no longer exists in the working directory._

Reply:
I think that's even better to remove it because the case should never be encountered due to some fixes I made in subsequent commits...

I will remove it.